File: ArtFragment/app/src/main/java/com/alierdemalkoc/artfragment/roomdb/ArtDatabase.java
package com.alierdemalkoc.artfragment.roomdb;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.alierdemalkoc.artfragment.model.ArtModel;

@Database(entities = {ArtModel.class},version = 1)
public abstract class ArtDatabase extends RoomDatabase {
    public abstract ArtDao artDao();
}

File: ArtBookTest/app/src/main/java/com/alierdemalkoc/artbooktest/roomdb/ArtDatabase.kt
package com.alierdemalkoc.artbooktest.roomdb

import androidx.room.Database
import androidx.room.RoomDatabase
import com.alierdemalkoc.artbooktest.model.Art

@Database(entities = [Art::class], version = 1)
abstract class ArtDatabase : RoomDatabase(){
    abstract fun artDao() : ArtDao
}